lib/acts_as_taggable_on/taggable/tagged_with_query/query_base.rb

```diff
@@ -48,7 +48,7 @@ def tags_match_type
 
     def escaped_tag(tag)
       tag = tag.downcase unless ActsAsTaggableOn.strict_case_match
-      ActsAsTaggableOn::Utils.escape_like(tag)
+      tag.gsub(/[!%_]/) { |x| '!' + x }
```

Comment:
This one should be crazily spec'ed :).

Reply:
It's just a rollback of https://github.com/mbleigh/acts-as-taggable-on/commit/cc7a2c097ee9eeb4417328986f0dd49112cb05b3, nothing I wrote for this PR. That commit breaks builds in master branch.

Reply:
Not to repeat myself, but this change is unrelated to the scope of this PR, as it barely rollbacks commit that broke the build here and in master branch